_data/navigation.yml

```diff
@@ -63,6 +63,10 @@ setup:
             url: /setup/install/providers/kubernetes/
           - title: Kubernetes (Manifest Based)
             url: /setup/install/providers/kubernetes-v2/
+            collapsed: false
+            children:
+              - title: Amazon EKS
+                url: /setup/install/providers/aws/aws-eks/
```

Comment:
This link should be under /setup/install/providers/kubernetes/eks rather than aws